spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc

```diff
@@ -68,45 +68,45 @@ image::dataflow-streams-list-definitions.png[List of Stream Definitions, scaledw
 
 The _Tasks_ section of the Dashboard currently has three tabs:
 
-* Modules
+* Apps
 * Definitions
 * Executions
 
-[[dashboard-tasks-modules]]
-=== Modules
+[[dashboard-tasks-apps]]
+=== Apps
 
-_Modules_ encapsulate a unit of work into a reusable component. Within the Dataflow
-runtime environment Modules allow users to create definitions for _Streams_ as
-well as _Tasks_. Consequently, the _Modules_ tab within the _Tasks_ section
+_Apps_ encapsulate a unit of work into a reusable component. Within the Dataflow
+runtime environment Apps allow users to create definitions for _Streams_ as
```

Comment:
I guess so, although in hindsight _this_ section _seems_ to _overuse_ italics _possibly_